cin_validator/tinkering.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
```

Comment:
Was this your "trying_things" file?
Did you intentionally add it to the commit?

Reply:
oops haha, I tend to do my comit as ./cin_validator so it adds everything